mne/viz/_brain/_brain.py

```diff
@@ -2316,6 +2316,41 @@ def add_label(self, label, color=None, alpha=1, scalar_thresh=None,
             self._labels[hemi].append(label)
         self._renderer._update()
 
+    @fill_doc
+    def add_head(self, color=None, alpha=0.5):
+        """Add a mesh to render the outer head surface.
+
+        Parameters
+        ----------
+        color : matplotlib-style color | None
+            A list of anything matplotlib accepts: string, RGB, hex, etc.
+            (default: "gray").
+        alpha : float in [0, 1]
+            Alpha level to control opacity.
+
+        Notes
+        -----
+        .. versionadded:: 0.24
+        """
+        from matplotlib.colors import colorConverter
+
+        # load seghead
+        seghead_fname = _check_fname(
+            op.join(self._subjects_dir, self._subject_id,
+                    'surf', 'lh.seghead'), overwrite='read', must_exist=True)
+        verts, triangles = read_surface(seghead_fname)
```

Comment:
This should be refactored with the code and API in `plot_alignment` to support multiple sources for the head as well as dense/sparse representations, etc. Sometimes people will want the dense surface, sometimes the sparse, and `plot_alignment` has some control over this (including a nice `'auto'` option that should be the default here).

Reply:
So is it worth making an intermediate step where `plot_alignment` and `mne.viz.Brain` share internal functions or should we just make one PR that makes `plot_alignment` a shallow wrapper for `mne.viz.Brain`?

Reply:
Let's just start with refactoring the head stuff. Refactoring it all is a much bigger undertaking. Piecemeal is much easier to review

Reply:
Ok one refactoring complete!